source/uwp/Renderer/lib/InputItem.cpp

```diff
@@ -260,3 +260,18 @@ void InputItem::Serialize(Json::Value & jsonValue)
         }
     }
 }
+
+std::string InputItem::InputItemCollectionAsJsonString(const std::vector<AdaptiveCards::XamlCardRenderer::InputItem>& inputElements)
```

Comment:
>InputItemCollectionAsJsonString [](start = 23, length = 31)

How would you feel about making this a member method of AdaptiveInputs?  That's the only place that uses it anyway, and it seems to fit in better with the charter of that class (it works on a group of inputs rather than a single one).

Reply:
I think I added it here believing the other places where we create the input json needed it, but it ended up not being the case. That makes total sense.

---
In reply to: [140822383](https://github.com/Microsoft/AdaptiveCards/pull/678#discussion_r140822383) [](ancestors = 140822383)